dev/com.ibm.ws.security.acme/src/com/ibm/ws/security/acme/internal/AcmeProviderImpl.java

```diff
@@ -290,7 +290,14 @@ public void revoke(List<X509Certificate> certificateChain, String reason) throws
 		acquireWriteLock();
 		try {
 			X509Certificate certificate = getLeafCertificate(certificateChain);
-			getAcmeClient().revoke(certificate, reason);
+			if (certificate == null) {
+				return;
+			}
+			String directoryURI = acmeHistory.getDirectoryURI(certificate.getSerialNumber().toString(16));
+			if (directoryURI == null) {
+				directoryURI = acmeConfig.getDirectoryURI();
```

Comment:
Probably some debug indicating the certificate wasn't found in the history file.